New content (#216)
archway-network/archway-docs: content/3.community/1.wallet.md

objectID: community|wallet
parentSection: Participate
parentSectionPath: /community

> title: Set up your Archway wallet

Reviewer: ```suggestion
title: Set up your archway wallet
```

> New wallets can be created in different ways, by using:

Reviewer: ```suggestion
New wallets can be created in different ways, including:
```

- [archwayd](/developers/getting-started/install#install-archwayd-using-docker) (archway core daemon)
- [Developer CLI](/developers/developer-tools/developer-cli)
- <a href="https://wallet.keplr.app/" target="_blank">Keplr</a>
- <a href="https://wallet.cosmostation.io/cosmos" target="_blank">Cosmostation</a>
- <a href="https://www.ledger.com/" target="_blank">Ledger</a>

::alert{variant="info"}

> Please note that 3rd party tools are not supported or maintained by Archway directly

Reviewer: ```suggestion
Please note that third-party tools are not supported or maintained directly by Archway
```

## Using a Browser Extension (Keplr)

### Creating an address with Keplr

> As previously stated, you will be able to derive your Archway address from your Cosmos address once Archway's mainnet is launched. 

Reviewer: ```suggestion
Your address for Archway's Testnets and Mainnet (when launched) will be automatically generated for your account once the respective network has been added to Keplr.
```

> [Keplr](https://wallet.keplr.app/) is a popular Cosmos wallet that is managed as a browser extension. For instructions regarding downloading, installing, and setting it up, you can have a look at Keplr's own [installation guide](https://keplr.crunch.help/getting-started/installing-keplr-wallet).

Reviewer: ```suggestion
[Keplr](https://wallet.keplr.app/) is a popular Cosmos wallet that is managed as a browser extension. For instructions on downloading, installing, and setting it up, see Keplr's [installation guide](https://keplr.crunch.help/getting-started/installing-keplr-wallet).
```

![](/images/docs/keplr09.png)

## Using a mobile app (Cosmostation)

### Creating an address with Cosmostation

> If you prefer having an mobile app wallet, you can use <a href="https://wallet.cosmostation.io/cosmos" target="_blank">Cosmostation</a>. You will be able to derive your Archway address from your Cosmos address once Archway's mainnet is launched. Cosmostation is available both on <a href="https://play.google.com/store/apps/details?id=wannabit.io.cosmostaion&pli=1">Google Play</a> and the <a href="https://apps.apple.com/app/cosmostation/id1459830339" target="_blank">Apple store</a>.

Reviewer: ```suggestion
If you prefer having a mobile app wallet, you can use <a href="https://wallet.cosmostation.io/cosmos" target="_blank">Cosmostation</a>. Your address for Archway's Testnets and Mainnet (when launched) will be automatically generated for your account once the respective network has been added to Cosmostation. Cosmostation is available both on <a href="https://play.google.com/store/apps/details?id=wannabit.io.cosmostaion&pli=1">Google Play</a> and the <a href="https://apps.apple.com/app/cosmostation/id1459830339" target="_blank">Apple store</a>.
```
